### DISCUSSION TITLE
Historical reason for color chart swatches be sorted that way

### DISCUSSION THREAD

**Discussion by ssh4net** [colour-science/colour#1288] [Q&A] [answered]
Hi, @KelSolaar.

What was a historical reason for having color chart patches sorted as they are stored in Colour-Science?

I found all official or babelcolor CGATK files sorted by patch names:
```
A1
A2
A3
A4
B1
..
```
Same for Digital SG.

When Colour-Science sorts patches, that letter is a column's name.

Is this going from some research papers?

**Comment by thomasmansencal** [comment]
Hello,

> When Colour-Science sorts patches, that letter is a column's name.

I'm not sure what you mean here? Are you saying that we are using labels vs 2 letter codes?

**Comment by ssh4net** [comment]
in CGATK files values stored as:
`A1, A2, A3, A4, B1, B2, B3, B4, ...`

In Colour-Science they are stored in
`A1, B1, C1, D1, E1, F1, A2, B2...`

and to use external data it need to 
`reshape(6,4,3).swapaxes(1,0)`

**Reply by thomasmansencal** [reply, marked as answer]
This is the order the patches are ordered in BabelColor spreadsheets:
![image](https://github.com/user-attachments/assets/48ad7acb-a7e0-4f95-9693-3fada6c6c265)

**Reply by thomasmansencal** [reply]
Note that for a human being, this is the order that kind of makes sense, e.g. one takes the last row and has all the neutral colours. Column wise makes this painful.

**Reply by ssh4net** [reply]
That's true :)
